var/spack/repos/builtin/packages/raja/package.py

```diff
@@ -123,32 +157,54 @@ class Raja(CachedCMakePackage, CudaPackage, ROCmPackage):
     )
 
     variant("openmp", default=True, description="Build OpenMP backend")
-    variant("shared", default=True, description="Build Shared Libs")
+    variant("shared", default=True, description="Build shared libs")
```

Comment:
Should these defaults be changed to match the defaults in RAJA CMake? Specifically, they are openmp - false, shared - false, desul - false, vectorization - false

Reply:
Shared, desul and vectorization almost certainly.  I didn't realize the OpenMP default was false now, is that a fallback to host sequential?

Reply:
We decided to make all parallel back-ends have the same default status.

Reply:
Done